### PR TITLE
Reformat unlimited.h

### PR DESCRIPTION
**Pull request by kyuupichan**
This is a reformat of unlimited.h with clang-format.

I will follow up with a .cpp file: unlimited.cpp, so we get a feel for one of each.

If we agree the clang-format is suitable then I will submit PRs for a reformatting of the other BU-specific files as they are not subject to merge conflict concerns.

I added the 120 column limit to .clang-format that was informally agreed in slack discussions

### CODE REVIEW

**Comment by AllanDoensen** [comment]
Is there any possibility of reformatting the BU specific GUI code as well? 
specifically:
 - src/qt/unlimiteddialog.cpp|h
 - src/qt/unlimitedmodel.cpp|h

**Comment by kyuupichan** [comment]
For sure.  @gandrewstone has indicated he will take care of the reformatting.

File: src/unlimited.h [hunk 1]
```diff
@@ -186,32 +199,33 @@ extern std::set<uint256> setUnVerifiedOrphanTxHash;
 extern CCriticalSection cs_xval;
 // Xpress Validation: end
 
-extern void LoadFilter(CNode *pfrom, CBloomFilter *filter);
+extern void LoadFilter(CNode* pfrom, CBloomFilter* filter);
```
Comment (marcusdiaz) [inline src/unlimited.h:203]: Per the review for the unlimited.cpp, then this also would need to be changed to have the star on the other side.

**Review by marcusdiaz — APPROVED** [review]
I approve irrespective the group decision on where to have clang position the dereference operator.

**Pull request closed without merging**